src/poprox_concepts/domain/click.py

```diff
@@ -7,4 +7,4 @@
 class Click(BaseModel):
     article_id: UUID
     newsletter_id: UUID | None = None
-    timestamp: datetime
+    timestamp: datetime | None = None
```

Comment:
In order to be used interchangeably with the old `ClickHistory`, the timestamp had to become optional. We'll still fill it in when we fetch clicks from the database but not necessarily when we use e.g. the MIND dataset in offline evaluations.